app.py

```diff
@@ -124,19 +131,26 @@ def authenticate():
         # If the request contains an email parameter provided by the user.
         if clientapp_decoded_token and request.is_json:
```

Comment:
if without else?

Reply:
Fixed. Although the else wasn't there because of the same level return.